AmazonChimeSDKDemo/AmazonChimeSDKDemo/LiveTranscriptionOptionsViewController.swift

```diff
@@ -139,31 +223,150 @@ class LiveTranscriptionOptionsViewController: UIViewController, UITextFieldDeleg
         engineTextField.inputView = enginePickerView
         languageTextField.inputView = languagePickerView
         regionTextField.inputView = regionPickerView
+        partialResultsStabilizationTextField.inputView = partialResultsStabilizationPickerView
+        piiContentIdentificationTextField.inputView = piiContentIdentificationPickerView
+        piiContentRedactionTextField.inputView = piiContentRedactionPickerView
         
         engineSelected = "transcribe"
         languageSelected = "en-US"
         regionSelected = "auto"
+        partialResultsStabilizationSelected = 0
+        piiContentIdentificationSelected = ""
+        piiContentRedactionSelected = ""
         
         engineTextField.text = enginesDict[engineSelected]
         languageTextField.text = languagesDict[languageSelected]
         regionTextField.text = regionsDict[regionSelected]
-        
+        partialResultsStabilizationTextField.text = partialResultsStabilizationsList[partialResultsStabilizationSelected]
+        piiContentIdentificationTextField.text = piiContentIdentificationDict[piiContentIdentificationSelected]
+        piiContentRedactionTextField.text = piiContentRedactionDict[piiContentRedactionSelected]
+    
         enginePickerView.delegate = self
         languagePickerView.delegate = self
         regionPickerView.delegate = self
         enginePickerView.dataSource = self
         languagePickerView.dataSource = self
         regionPickerView.dataSource = self
+        partialResultsStabilizationPickerView.delegate = self
+        partialResultsStabilizationPickerView.dataSource = self
+        piiContentIdentificationPickerView.delegate = self
+        piiContentIdentificationPickerView.dataSource = self
+        piiContentRedactionPickerView.delegate = self
+        piiContentRedactionPickerView.dataSource = self
         
         enginePickerView.tag = 1
         languagePickerView.tag = 2
         regionPickerView.tag = 3
+        partialResultsStabilizationPickerView.tag = 4
+        piiContentIdentificationPickerView.tag = 5
+        piiContentRedactionPickerView.tag = 6
+        
+        storyBoard = UIStoryboard(name: "Main", bundle: nil)
+        languageDetectionViewController = storyBoard!.instantiateViewController(withIdentifier: "languageDetection") as? AutomaticLanguageDetectionViewController
```

Comment:
same so it is usually preferred to be
```
guard languageDetectionViewController = storyBoard?.instantiateViewController(withIdentifier: "languageDetection") as? AutomaticLanguageDetectionViewController else {
      // Maybe some warning.
      return
}
````

Reply:
@hokyungh so I have this initialized in viewDidLoad() as 
```
languageOptionsViewController =
                storyBoard?.instantiateViewController(withIdentifier: "languageOptions")
                as? AutomaticLanguageOptionsViewController
```
I have a function that shows a new screen for automatic language identification when a toggle switch is pressed 
```
@IBAction func showLanguageOptionsView(_ sender: UISwitch) {
        if sender.isOn {
            languageHandler?(transcribeLanguages)
            languagesDictHandler?(languagesDict)
            languageOptionsViewController?.modalPresentationStyle = .pageSheet
            self.present(languageOptionsViewController!, animated: true, completion: nil)
        }
    }```
After I added `guard let languageDetectionViewController` the above function still makes it to be null. How do I handle this?

Reply:
ignore... got it figured out in a later commit and updated.